components/formats-gpl/pom.xml

```diff
@@ -59,7 +59,7 @@
       <version>${project.version}</version>
     </dependency>
     <dependency>
-      <groupId>ome</groupId>
+      <groupId>idr</groupId>
```

Comment:
This needs to be reverted (and maybe accounted for in the script `groupId` functionality)